meson.build

```diff
@@ -16,6 +16,8 @@ project(
     'c_args=-Wno-unused-function -Wno-conversion -Wno-misleading-indentation -Wno-incompatible-pointer-types',
     'fortran_args=-Wno-conversion',
     'fortran_std=legacy',
+    'blaslib=openblas',
+    'lapacklib=openblas'
```

Comment:
Looks good, almost there. Can I ask for one minor tweak - naming it `blas`/`lapack` instead of `blaslib`/`lapacklib`?

Reply:
Sure, will do that